### PR TITLE
Add scikit-fmm

### PR DESCRIPTION
**Pull request by jakirkham**
A very fast marching method.

### CODE REVIEW

File: scikit-fmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - python
+    - numpy
+
+test:
+  imports:
+    - skfmm
+
+about:
+  home: https://github.com/scikit-fmm/
+  license: BSD
```
Comment (ccordoba12) [inline scikit-fmm/meta.yaml]: Could you add a `summary` please?

File: scikit-fmm/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
+
+:: Add more build steps here, if they are necessary.
+
+:: See
+:: http://docs.continuum.io/conda/build.html
+:: for a list of environment variables that are set during the build process.
```
Comment (ccordoba12) [inline scikit-fmm/bld.bat]: Please remove these comments

**Comment by jakirkham** [comment]
Thanks for the feedback, @ccordoba12. I have tried to address your criticisms. Please let me know what you think.

File: scikit-fmm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    - python
+    - numpy
+
+test:
+  imports:
+    - skfmm
+
+about:
+  home: https://github.com/scikit-fmm/
+  summary: An extension module implimenting the fast marching method
```
Comment (ccordoba12) [inline scikit-fmm/meta.yaml]: Minor typo here: `implimenting` -> `implementing`

Comment (jakirkham) [inline scikit-fmm/meta.yaml]: Yeah, I copied this from the package description, but I will fix it.

Comment (jakirkham) [inline scikit-fmm/meta.yaml]: Fixed.

File: scikit-fmm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: scikit-fmm
+  version: 0.0.7
+
+source:
+  fn: scikit-fmm-0.0.7.zip
+  url: https://pypi.python.org/packages/source/s/scikit-fmm/scikit-fmm-0.0.7.zip#md5=c669e98b1b279ce9c4df1866df6a2370
```
Comment (ccordoba12) [inline scikit-fmm/meta.yaml]: There's no need for the url to contain the `md5` part. Please remove it.

Comment (jakirkham) [inline scikit-fmm/meta.yaml]: Somehow forgot about that. Now removed.

**Comment by jakirkham** [comment]
Any more thoughts, @ccordoba12?

**Comment by ccordoba12** [comment]
Nop. Thanks for your work, merging :-)

**Pull request merged**